public/app/plugins/datasource/graphite/datasource.ts

```diff
@@ -699,3 +704,17 @@ function supportsTags(version: string): boolean {
 function supportsFunctionIndex(version: string): boolean {
   return isVersionGtOrEq(version, '1.1');
 }
+
+function mapToTags(): OperatorFunction<any, Array<{ text: string }>> {
```

Comment:
@hugohaggmark @torkelo do you know if we have any interface or types for graphite tags?

Reply:
Not sure